src/components/DataSession/OperationWizard.vue

```diff
@@ -215,6 +215,7 @@ const handleThumbnailClick = (item) => {
 <style scoped>
 .wizard-background{
   background-color: var(--dark-blue);
+  height: 100vh;
```

Comment:
The buttons are in the right place but from the video it still looks like the wizard component is overflowing and we have to scroll. For the tablet we don't want to have to scroll at all but want to see everything on one screen

Reply:
wasn't what I had in mind so actually we can go back to how it was before